core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -639,13 +639,13 @@ portfolio.pending.autoConf.state.ERROR=An error at a service request occurred. N
 # suppress inspection "UnusedProperty"
 portfolio.pending.autoConf.state.FAILED=A service returned with a failure. No auto-confirm possible.
 
-portfolio.pending.step1.info=Deposit transaction has been published.\n{0} need to wait for at least one blockchain confirmation before starting the payment.
+portfolio.pending.step1.info=Deposit transaction has been published.\n{0} needs to wait for 10 confirmations (about 20 minutes) before the payment can start.
```

Comment:
This line causes "You needs to wait...".

Guessing it says "The XMR buyer needs to wait" if they're the seller.

We can re-word it or pass in the right text, including "need" or "needs", into {0} where it's called in the code.

Reply:
Reverted to "need" as it was done initially.